src/components/InstanceFilters/InstanceFilters.js

```diff
@@ -202,65 +223,16 @@ const InstanceFilters = props => {
     return _.orderBy(options, ['count'], ['desc']);
   };
 
-  const effectiveLocationOptions = locations.map(({ name, id }) => ({
-    label: name,
-    value: id,
-  }));
-
-  const resourceTypeOptions = resourceTypes.map(({ name, id }) => ({
-    label: name,
-    value: id,
-  }));
-
-  const instanceFormatOptions = instanceFormats.map(({ name, id }) => ({
-    label: name,
-    value: id,
-  }));
-
-  const modeOfIssuanceOptions = modesOfIssuance.map(({ name, id }) => ({
-    label: name,
-    value: id,
-  }));
-
-  const natureOfContentOptions = natureOfContentTerms.map(({ name, id }) => ({
-    label: name,
-    value: id,
-  }));
-
-  const suppressedOptions = [
-    {
-      label: <FormattedMessage id="ui-inventory.yes" />,
-      value: 'true',
-    },
-    {
-      label: <FormattedMessage id="ui-inventory.no" />,
-      value: 'false',
-    },
-  ];
-
-  const sourceOptions = [
-    {
-      label: <FormattedMessage id="ui-inventory.folio" />,
-      value: 'FOLIO',
-    },
-    {
-      label: <FormattedMessage id="ui-inventory.marc" />,
-      value: 'MARC',
-    },
-  ];
-
-  const langOptions = languageOptions(intl);
-
   const [facetsOptions, setFacetsOptions] = useState({
-    effectiveLocationOptions,
-    langOptions,
-    resourceTypeOptions,
-    instanceFormatOptions,
-    modeOfIssuanceOptions,
-    natureOfContentOptions,
-    suppressedOptions,
-    sourceOptions,
-    tagsRecords,
+    effectiveLocationOptions: [],
+    langOptions: [],
+    resourceTypeOptions: [],
+    instanceFormatOptions: [],
+    modeOfIssuanceOptions: [],
+    natureOfContentOptions: [],
+    suppressedOptions: [],
```

Comment:
We need to split the suppressedOptions into staffSurpress and suppressDiscovery

Reply:
Thanks, fixed.